lsp/core/aws-lsp-core/src/credentials/updateCredentialsPayload.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Payload for custom notification "Update Credentials"
+ * Used by the host to send resolved credentials
+ */
+export interface UpdateCredentialsPayload {
```

Comment:
question: Maybe not for PoC, but does it make sense to put all of the comm interfaces written in Typescript into a shared library so the LSP, VSCode, and whatever other TS/JS consumers can reuse them?

Reply:
That could be a worthwhile north star. @abailiss and I were discussing something similar.